src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationProcessingServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.Collection;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseDetailedReportDTO;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationService;
+
+public class ConcurEventNotificationProcessingServiceImpl implements ConcurEventNotificationProcessingService {
+    
+    protected ConcurEventNotificationService concurEventNotificationService;
+    protected ConcurAccountValidationService concurAccountValidationService;
+
+    @Override
+    public void processConcurEventNotifications() {
+        Collection<ConcurEventNotification> concurEventNotifications = concurEventNotificationService.retrieveConcurEventNotificationsForProcessing();
+
+        if (concurEventNotifications != null && CollectionUtils.isNotEmpty(concurEventNotifications)) {
+            for (ConcurEventNotification concurEventNotification : concurEventNotifications) {
+                ExpenseDetailedReportDTO expenseDetailedReport = retrieveExpenseDetailedReportFromConcur(concurEventNotification);
+                ConcurAccountInfo concurAccountInfo = extractAccountInfo(expenseDetailedReport);
+                concurAccountValidationService.validateConcurAccountInfo(concurAccountInfo);
+                updateExpenseReportStatusInConcur();
+            }
+        }
+    }
+
+    @Override
+    public ExpenseDetailedReportDTO retrieveExpenseDetailedReportFromConcur(
```

Comment:
It looks a little odd to me to have the method signatures of these methods split across two lines. They don't seem too long to be on one line, but maybe that's just personal preference.